### DISCUSSION TITLE
Does garbo usually try to get something from the shore gift shop?

### DISCUSSION THREAD

**Discussion by cpenny86** [loathers/garbage-collector#1656] [Q&A] [answered]
I started running garbo this morning, previous to the trivial KoL update that has made the shore gift shop a separate map location (which is a great change). Garbo keeps entering the shore and hanging because there are only vacation options there now -- I didn't realize what had happened at first and just clicked out of it, and then it did it again. 

Obviously I don't expect the code to be updated for a game change that's like half an hour old! But does anyone know what it's trying to get from the gift shop? I am happy to grab something there manually if it makes the script happy, but I can't remember seeing it do that before. Meat globe, maybe?

**Comment by pstalcup** [comment, marked as answer]
Garbo was using the shore in a way to help with Extrovermectin copies. This has been changed and garbo has been updated. 
It never actually bought anything from the shore.
